config.yaml

```diff
@@ -62,4 +62,4 @@ logfile_log_pattern: "{({d(%Y-%m-%d %H:%M:%S)} [{l}]):26.26} {m}{n}"
 
 # More detailed log patterns
 #console_log_pattern: "{d(%H:%M:%S.%3f%z)} [{h({l}):<5}] [{T}] [{t}] - {M}:{m}{n}"
-#logfile_log_pattern: "{d(%Y-%m-%dT%H:%M:%S.%3f%z)} [{h({l}):<5}] [{T}]-[{t}] [{f}:{L}] - {M}:{m}{n}"
\ No newline at end of file
+#logfile_log_pattern: "{d(%Y-%m-%dT%H:%M:%S.%3f%z)} [{h({l}):<5}] [{T}]-[{t}] [{f}:{L}] - {M}:{m}{n}"
```

Comment:
what was changed here?

Reply:
I think github removed whitespace..but not sure

Reply:
This is likely a line-ending change. This needs to be pulled, changed to linux line endings, and recommitted. @shefass can you do this?